src/View/SerializedView.php

```diff
@@ -84,8 +85,23 @@ public function render($view = null, $layout = null)
         }
 
         if ($serialize !== false) {
-            return $this->_serialize($serialize);
-        } elseif ($view !== false && $this->_getViewFileName($view)) {
+            $result = $this->_serialize($serialize);
+            if ($result === false) {
+                if (Configure::read('debug') && in_array('trace', $serialize)) {
```

Comment:
I suspect there was something unserializable in the trace. I wasn't able to reproduce this error in bare-bones application I have setup.